app/modbus.mjs

```diff
@@ -32,25 +32,26 @@ const AVAILABLE_SETTINGS = {
-    3: { name: 'TE10InletAfterHeaterHot', description: 'TE10 inlet after heater hot' }, // 3=TE10 Tulo lämmityspatterin jälkeen kuuma
-    4: { name: 'TE20RoomHot', description: 'TE20 room hot' }, // 4=TE20 Huone kuuma
-    5: { name: 'TE30OutletCold', description: 'TE30 outlet cold' }, // 5=TE30 Poisto kylmä
-    6: { name: 'TE30OutletHot', description: 'TE30 outlet hot' }, // 6=TE30 Poisto kuuma
-    7: { name: 'HPFault', description: 'HP fault' }, // 7=HP vika
-    8: { name: 'HeaterFault', description: 'Heater fault' }, // 8=SLP vika
-    9: { name: 'ReturnWaterCold', description: 'Return water cold' }, // 9=Paluuvesi kylmää
-    10: { name: 'LTOFault', description: 'Heatexchanger fault' }, // 10=LTO vika
-    11: { name: 'CoolingFault', description: 'Cooling fault' }, // 11=Jäähdytys vika
-    12: { name: 'EmergencyStop', description: 'Emergency stop' }, // 12=Hätäseis
-    13: { name: 'FireRisk', description: 'Fire risk' }, // 13=Palovaara
-    14: { name: 'ServiceReminder', description: 'Service reminder' }, // 14=Huoltomuistutus
-    15: { name: 'HeaterPressureSwitch', description: 'Heater pressure switch' }, // 15=SLP painevahti
-    16: { name: 'InletFilterDirty', description: 'Inlet filter dirty' }, // 16=Tulosuodatin likainen,
-    17: { name: 'OutletFilterDirty', description: 'Outlet filter dirty' }, // 17=Poistosuodatin likainen
-    20: { name: 'InletFanPressureAbnomaly', description: 'Inlet fan pressure abnomaly' }, // 20=Tulopuhallin painepoikkeama
-    21: { name: 'OutletFanPressureAbnomaly', description: 'Outlet fan pressure abnomaly' }, // 21=Poistopuhallin painepoikkeama
+    // Alarm number
+    1: { name: 'TE5SupplyAirAfterHRCold', description: 'TE5 Supply air after HR cold' },
+    2: { name: 'TE10SupplyAirAfterHeaterCold', description: 'TE10 Supply air after heater cold' },
+    3: { name: 'TE10SupplyAirAfterHeaterHot', description: 'TE10 Supply air after heater hot' },
+    4: { name: 'TE20RoomTempHot', description: 'TE20 Room temp hot' },
+    5: { name: 'TE30ExtractAirCold', description: 'TE30 Extract air cold' },
+    6: { name: 'TE30ExtractAirHot', description: 'TE30 Extract air hot' },
+    7: { name: 'HPError', description: 'Heatpump error' },
+    8: { name: 'EHError', description: 'Heater error' }, // Electrical Heater error?
+    9: { name: 'ReturnWaterCold', description: 'Return water cold' },
+    10: { name: 'HRError', description: 'Heat recovery error' },
+    11: { name: 'CoolingError', description: 'Cooling error' },
+    12: { name: 'EmergencyStop', description: 'Emergency stop' },
+    13: { name: 'FireRisk', description: 'Fire risk' },
+    14: { name: 'ServiceReminder', description: 'Service reminder' },
+    15: { name: 'EHPDA', description: 'Heater pressure switch' }, // ?? EH PDA / SLP painevahti
+    16: { name: 'SupplyFilterDirty', description: 'Supply filter dirty' },
+    17: { name: 'ExtractFilterDirty', description: 'Extract filter dirty' },
```

Comment:
We use "exhaust" in the rest of the app

Reply:
Extract is outgoing - before the heat exchange - and exhaust is what's leaving to the outside.

Reply:
The air that leaves the house is called "waste air" in other places 🤨

Reply:
<img width="328" alt="Enervent_airducts_naming" src="https://user-images.githubusercontent.com/3820361/153711085-e33eb479-5d52-49d3-9d4d-d0607492f0ed.png">
Yes, I aslo mentioned this in the Alarm name issue.
Maybe the Alarm-code should have the Enervent naming - but the description could use Waste Air..  Which means that we can probably change it later if we want.
I like Waste Air better.

Reply:
Interesting
> Maybe the Alarm-code should have the Enervent naming - but the description could use Waste Air.
This sounds like a good compromise

Reply:
This got complicated now 😄 The Finnish comment referred to "poistoilma" which is exhaust air, not waste air.

Reply:
I had a look at the English Modbus registers again and I believe the alarm descriptions are simply wrong. For example, register 3x0015 is named "Exhaust air filter Pa", not "extract air".

Also, it makes no sense to have an alarm for "Waste air too cold" because you want it to pretty much be as cold as possible (that means you extracted more heat from it).

All in all, I believe the TE30 alarms should look like this:

```
    5: { name: 'TE30ExhaustAirCold', description: 'TE30 Exhaust air cold' },
    6: { name: 'TE30ExhaustAirHot', description: 'TE30 Exhaust air hot' },
```

Reply:
The dimension drawings image also talks about "Outside air" while the register list talks about "Fresh air", so I believe we should discard the dimension drawings and stick to what the register list says. The only mentions of "extract" in the register list is the alarm codes, and like I said in the previous comment I think that's just a typo and they mean "exhaust".

Reply:
Changed these to extract air - as that is what it checks for. Exhaust-air has no checks. (as concluded in the issue).
Exhaust filter Pa seems wrong as there is no filter after the Heat recovery. But I chose to keep the name Waste here - if we use the waste - name for all outgoing air (no matter if it's before or after the HRC), so it's more "the filter on the air going out"-alert.  Which should make it obvious to anyone what we mean.
But it is funny that all these words seems to be used in different ways  even by the same manufacturer  depending on when or by who that wrote the docs.
Fresh or Outside is for sure the same thing. :-). Extract/Exhaust seems to be the most commonly used name when talking about Heat recovery Ventilation systems. Maybe Waste is used more when there is no heat recovery (same pipe and same values the whole way out).